### DISCUSSION TITLE
Does anyone have a working example of ResourceList filters?

### DISCUSSION THREAD

**Discussion by jamieroyce** [Shopify/polaris-react#3187] [General]
I am trying to follow the documentation for the Polaris ResourceList filter and can't figure out how to get this to work. It would be really helpful to show how this works with actual data. I'm using GraphQL and set up my app following the react node tutorial.

**Comment by mcjambi** [comment]
You can check https://github.com/Shopify/product-reviews-sample-app, they use ResourceList with filter!
